x/gov/keeper/proposal_test.go

```diff
@@ -244,6 +288,13 @@ func (suite *KeeperTestSuite) TestCancelProposal() {
 			}
 		})
 	}
+	_, err = suite.govKeeper.Votes.Get(suite.ctx, collections.Join(proposalID, suite.addrs[0]))
+	suite.Require().ErrorContains(err, collections.ErrNotFound.Error())
+
+	// check that proposal 3 votes are still present in the state
```

Comment:
Shouldn't the votes length be empty?

Reply:
This just makes sure the votes for other proposals which doesn't go through CancelProposal still persists in state

Reply:
I see, I didn't see that proposal 3 still had votes and wasn't cancelled. Maybe a comment above proposal3 should precise it is only here for what you just explained?